browserstack.gradle

```
@@ -0,0 +1,29 @@
+import groovy.json.JsonSlurper
+
+/**
+ * This task will delete all previously uploaded AppLive apps for the provided customId
+ */
+ext.deleteAppLiveWithCustomId = { customId ->
+    def response = new ByteArrayOutputStream()
+    exec {
+        executable "curl";
+        args "-u", "$rootProject.browserStackUser:$rootProject.browserStackKey", "-X",
+                "GET", "https://api-cloud.browserstack.com/app-live/recent_apps/${customId}"
+        standardOutput = response;
+    }
+    def json = new JsonSlurper().parseText(response.toString())
+    if (!(json instanceof List<Map>)) {
+        logger.lifecycle("[BROWSERSTACK] No apps to delete for customId[${customId}]")
```

If we don't parse error messages we should mention this in a log.

```suggestion
        logger.lifecycle("[BROWSERSTACK] No apps to delete for customId[${customId}] or error response was received from a server")
```

---

Added the json response to the log message.